src/Microsoft.Health.Fhir.R4.Web/appsettings.json

```diff
@@ -39,7 +39,10 @@
         },
         "Operations": {
             "Export": {
-                "Enabled": false,
+                "Enabled": true,
+                "StorageAccountType": null,
```

Comment:
You could simplify even further by remove this and only support blob storage in the Azure implementation.

Reply:
Deleted this config and related classes that depended on this flow. Also moved the `InMemoryExportDestinationClient` to the tests project